src/assets/create.js

```diff
@@ -19,11 +19,9 @@ class UI {
          </div>
       `;
 
-      document.querySelectorAll('.input').forEach((checked, i) => {
-        const taskName = checked.nextElementSibling;
-        checked.addEventListener('change', () => {
+      document.querySelectorAll('.input').forEach((complete, i) => {
+        complete.addEventListener('change', () => {
```

Comment:
- [ ] The completed value of the last task does not change upon user actions. It would be great to correct that to improve the user experience. 

<img width="900" alt="image" src="https://user-images.githubusercontent.com/77190203/185598052-042f76b7-4631-4726-854f-3a6b9c6c734c.png">